### DISCUSSION TITLE
[Keycloak-X] Is it possible to read KEYCLOAK_ADMIN and KEYCLOAK_ADMIN_PASSWORD from a file ?

### DISCUSSION THREAD

**Discussion by Leletir** [keycloak/keycloak#9362] [Q&A] [answered]
Hi,

First of all, thank you for this software !! It rocks !

I would like to know if it's possible to fetch the values of KEYCLOAK_ADMIN and KEYCLOAK_ADMIN_PASSWORD from a file ?

Our use case is that we use [Hashicorp Vault Injector](https://www.vaultproject.io/docs/platform/k8s/injector) to fetch our secrets from our vault to our pods.
Thanks to the Vault Injetor we obtain a file under `/vault/secrets`, for example:

/vault/secrets/admin-credentials

```shell
KEYCLOAK_ADMIN=toto
KEYCLOAK_ADMIN_PASSWORD=test
```

I would like to know if it's possible for Keycloak-X to read those files and use those key-values ? 

I've seen there is --vault-file-path option in build command, but I couldn't find documentation on how use it/which is the format of this file.

**Comment by vmuzikar** [comment, marked as answer]
@Leletir Thank you for raising this. Could you please move your question to https://github.com/keycloak/keycloak/discussions/8549 to have the discussion in a single place?

**Comment by Leletir** [comment]
Hello @vmuzikar, thank you for pointing #8549 I missed it when I opened this issue.

Have a good day.
